handlers/report/report.go

```diff
@@ -15,9 +15,22 @@ const (
 	LON_PARAM = "lon"
 )
 
-// weatherReportResponse JSON response for GET /weather-report
+// weatherReportResponse JSON response for GET
 type weatherReportResponse struct {
-	Data models.CurrentWeather `json:"data"`
+	Data currentWeather `json:"data"`
+}
+
+// currentWeather JSON response for current weather summary
+type currentWeather struct {
+	Latitude        float64 `json:"latitude"`
+	Longitude       float64 `json:"longitude"`
+	Condition       string  `json:"condition"`
+	TempDescription string  `json:"temp_description"`
+}
+
+// FromModel converts a CurrentWeather model to the currentWeather presentataion struct
+func FromModel(model models.CurrentWeather) currentWeather {
+	return currentWeather(model)
```

Comment:
It probably seems unnecessary to define a separate function for casting one struct to another. 

Relying on always being able to cast one struct to another tightly couples the model and presentation structs though in a way we don't want. 

We can convert by casting here since they're simple structs with the exact same attributes, but we don't want to bake in that limitation in general.